runtime/queries/bash/highlights.scm

```diff
@@ -1,110 +1,157 @@
-(simple_expansion) @none
-(expansion
-  "${" @punctuation.special
-  "}" @punctuation.special) @none
 [
- "("
- ")"
- "(("
- "))"
- "{"
- "}"
- "["
- "]"
- "[["
- "]]"
- ] @punctuation.bracket
+  "("
```

Comment:
Since we're directly upstreaming these queries (including their comments and whitespace) perhaps it would be worth it to document that here.

Reply:
What exactly would you like to see documented, and where?

Reply:
That would defeat the purpose of copying them verbatim, including comments and whitespace ;)

Reply:
(These are not vendored in the same sense; rather nvim-treesitter is the integration project preparing these files for us.)

Reply:
Fair enough.